http/responses/main_test.go

```diff
@@ -502,11 +503,11 @@ func TestInternalError(t *testing.T) {
 			res := tt.httpTestShit.recorder.Result()
 
 			if res.StatusCode != tt.wantStatus {
-				t.Errorf("Created response invalid status code. want %d, got %d", tt.wantStatus, res.StatusCode)
+				assert.Equal(t, tt.wantStatus, res.StatusCode)
```

Comment:
The whole if statement can be replaced with the assertion. However, other parts of the file remain unchanged. Perhaps leave it for some other PR where all occurrences will be replaced?

Reply:
Removed